Boards.h

```diff
@@ -665,22 +665,29 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_PWM(p)           PIN_TO_DIGITAL(p)
 #define PIN_TO_SERVO(p)         ((p) - 2)
 
-// ESP8266 generic
+// ESP8266
```

Comment:
There are several different popular boards using this chip. We should include a comment that states which specific boards have been tested to work with this set of pin definitions.